src/Bigtable/Admin/V2/BigtableInstanceAdminClient.php

```diff
@@ -1,12 +1,12 @@
 <?php
 /*
- * Copyright 2017, Google LLC All rights reserved.
+ * Copyright 2018 Google LLC
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
  *
- *     http://www.apache.org/licenses/LICENSE-2.0
+ *     https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Why this change? I think you should keep it http.

Reply:
As @dwsupplee mentioned, this has been updated in toolkit for all languages. The reason for the update is to match Google's recommended open source license header.

Reply:
Ok, but https://www.apache.org/licenses/LICENSE-2.0 shows `http` URL. Where can I learn more about why we should use `https`?

 However, I'm cool if our lawyers are ok with the changes for the URL scheme.

But for the copyright year, I'm pretty sure you shouldn't change it.

Reply:
@tmatsuo, pubsub and logging GAPICs were originally introduced back in 2016 but due to refreshes their copyrights have been since updated. Are you in favor or me updating them to reflect their original introduction?

Reply:
Not sure about the `https` change, but if it was done in Toolkit then this PR is not the place to debate it. Looks like that was updated in https://github.com/googleapis/toolkit/pull/1748, so @lukesneeringer can speak to why it was changed.

As for the copyright year, we should update older GAPIC files to reflect the year they were introduced (e.g. 2016).